gravitee-apim-console-webui/src/services-ngx/group.service.ts

```diff
@@ -51,7 +53,49 @@ export class GroupService {
     return this.http.post<void>(`${this.constants.env.baseURL}/configuration/groups/${groupId}/members`, groupMembershipToSend);
   }
 
+  delete(id: string) {
+    if (id) {
+      return this.http.delete<void>(`${this.constants.env.baseURL}/configuration/groups/${id}`);
+    }
+  }
+
```

Comment:
I think we can assume that the id will always be truthy, so I would keep this simple: 
```suggestion
  delete(id: string) {
      return this.http.delete<void>(`${this.constants.env.baseURL}/configuration/groups/${id}`);
  }

```
WDYT?